app/api/sync/processNamespaces.ts

```diff
@@ -0,0 +1,353 @@
+/* eslint-disable max-lines */
```

Comment:
Instead of modifying this file, we opted for importing the models directly, now that the processNamespaces file is abstracted by model.  Still, some legacy namespaces still use this global "models" object, but none of those is processing the data, just syncing it directly, so that should not be an issue.